Software/Source/TotalPktCnt.py

```diff
@@ -0,0 +1,40 @@
+import json
+def check():
+    zero=0
+    one=0
+    two=0
+    three=0
+    four=0
+    f=open("0081F9F04991.txt","r")
+    while True:
+        content=f.readline()
+        if content:
+            data=json.loads(content)
+            check=data["PacketType"]
+            if(check=='0'):
+                zero=zero+1

```

Comment:
countPktType0+=1            //is recommened //Same as countPktType0=countPktType0+1